osquery/tables/system/windows/certificates.cpp

```diff
@@ -542,12 +688,37 @@ BOOL WINAPI certEnumSystemStoreLocationsCallback(LPCWSTR storeLocation,
   return TRUE;
 }
 
-QueryData genCerts(QueryContext& context) {
-  QueryData results;
+/**
+ * A user's `Personal` certs are stored on disk and not in the registry.
+ * Furthermore, when using the enumeration APIs, other users' Personal certs
+ * are not visible. This function proactively retrieves these certs from
+ * disk so that the table is guaranteed to, at the very least, list any
+ * existing Personal certs for all local users, regardless of whether those
+ * users' registry hives are currently mounted.
+ */
+void genPersonalCertsFromDisk(QueryData& results) {
+  auto users = SQL::selectAllFrom("users");
+  for (const auto& row : users) {
+    auto sid = row.at("uuid");
+    auto username = row.at("username");
+
+    findUserPersonalCertsOnDisk(username,
+                                sid,
+                                sid,
+                                "Personal", // storeName
+                                "Users", // storeLocation
```

Comment:
nit: not sure if this has changed or not, but I know in the past we prefer to avoid having comments on lines with code. I'd say maybe just remove these? Up to you.